lib/perl/Genome/Config/AnalysisProject/Command/Release.pm

```diff
@@ -46,7 +46,7 @@ sub __errors__ {
     my $self = shift;
     my @errors = $self->SUPER::__errors__(@_);
     for my $status (map{$_->status} $self->analysis_projects){
-        unless(grep{$_ eq $status} ("Pending", "Hold", "In Progress")){
+        unless(grep{$_ eq $status} ("Pending", "Hold", "In Progress","Deprecated")){
```

Comment:
To verify: we want to **allow** setting the status back to `Hold` or `In Progress` from `Deprecated`?

Either way, we may want to put a similar status check on the new command to prevent deprecating AnPs that are in the various "final" statuses (like `Purged` or `Template`).